Azure/azure-cli
# [RDBMS] `az postgres flexible-server fast-create`: Add fast create server for postgres

**Related command**
`az postgres flexible-server fast-create`

**Description**
This PR enables fast provisioning FSPG servers with pre-canned servers.

**Testing Guide**
`az postgres flexible-server fast-create -g testgroup -l westus --admin-user username --admin-password password --version 12 --storage-size 32 --tier Burstable --sku-name Standard_B1ms`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
